cmd/mayactl/app/command/volume_info.go

```diff
@@ -0,0 +1,102 @@
+package command
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	client "github.com/openebs/maya/pkg/client/jiva"
+	"github.com/openebs/maya/pkg/util"
+	"github.com/openebs/maya/types/v1"
+	"github.com/spf13/cobra"
+)
+
+var (
+	volumeInfoCommandHelpText = `
+	    Usage: mayactl volume info --volname <vol>
+
+            This command queries the info of the volume.
```

Comment:
This command fetches the information and status of the various aspects of the Volume like iSCSI, Controller and Replica.

Reply:
What kind of info does it fetch. Mentioning them will help. Please describe these elaborately.